### ISSUE TITLE
DataExporter: XLS postProcessor fails

### ISSUE THREAD

**Issue by Rapster** [primefaces/primefaces#10027]
### Describe the bug

Based on the snapshot showcase: http://localhost:8080/showcase/ui/data/dataexporter/customizedDocuments.xhtml I have this exception when I click on XLS:

```
AVERTISSEMENT: /ui/data/dataexporter/customizedDocuments.xhtml @24,96 postProcessor="#{customizedDocumentsView.postProcessXLS}": java.lang.NullPointerException
javax.el.ELException: /ui/data/dataexporter/customizedDocuments.xhtml @24,96 postProcessor="#{customizedDocumentsView.postProcessXLS}": java.lang.NullPointerException
	at com.sun.faces.facelets.el.TagMethodExpression.invoke(TagMethodExpression.java:71)
	at org.primefaces.component.datatable.export.DataTableExcelExporter.postExport(DataTableExcelExporter.java:93)
	at org.primefaces.component.datatable.export.DataTableExcelXStreamExporter.postExport(DataTableExcelXStreamExporter.java:56)
	at org.primefaces.component.datatable.export.DataTableExporter.export(DataTableExporter.java:211)
	at org.primefaces.component.export.DataExporter.processAction(DataExporter.java:171)
	at javax.faces.event.ActionEvent.processListener(ActionEvent.java:72)
	at javax.faces.component.UIComponentBase.broadcast(UIComponentBase.java:495)
	at javax.faces.component.UICommand.broadcast(UICommand.java:211)
	at javax.faces.component.UIViewRoot.broadcastEvents(UIViewRoot.java:847)
	at javax.faces.component.UIViewRoot.processApplication(UIViewRoot.java:1396)
	at com.sun.faces.lifecycle.InvokeApplicationPhase.execute(InvokeApplicationPhase.java:58)
```

I believe this might be a regression of #9290. SXSSFSheet has some sort of cache on rows, once you exceed the initial capacity it flushes the first row (e.g 0), so whenever you do` sheet.getRow(0)` it returns null

I'll most probably fix it with a PR for #9969 (My Plan was to make it avaible for 14.0.0 but depending what you feel it could be merged for 13.0.0)  

### PrimeFaces edition

None

### PrimeFaces version

13.0.0-SNAPSHOT

### JSF implementation

Mojarra

### JSF version

2.3

### Java version

8

**Comment by melloware** [comment]
Is there an interim smaller fix we can do for 13.0.0 ?

**Comment by Rapster** [comment]
Yes there is, I'll make a small PR for it
